SRC/dlassq.f90

```diff
@@ -189,12 +200,14 @@ subroutine DLASSQ( n, x, incx, scl, sumsq )
    if( sumsq > zero ) then
       ax = scl*sqrt( sumsq )
       if (ax > tbig) then
-         abig = abig + (ax*sbig)**2
+!        We assume scl >= sqrt( TINY*EPS ) / sbig
+         abig = abig + (scl*sbig)**2 * sumsq
          notbig = .false.
```

Comment:
Not an issue but I think that setting `notbig` here is unnecessary, it's not read again.

Reply:
Indeed. Thanks for noticing that!